rna-germline-variant-calling.wdl

```diff
@@ -732,6 +867,54 @@ task ScatterIntervalList {
     }
 }
 
+task ScatterIntervalList_GATK4 {
```

Comment:
Is there a reason to use the GATK4 version of this tool rather than Picard? I would think the Picard version is going to be kept more up to date for this particular tool.

Reply:
Its to exemplify that users can use Picard Tools in GATK. The original task used the Picard jar in the gitc docker which may not get updated that often either.